File: plugin/NoLinkify/src/main/java/NoLinkify.java
import android.content.Context;
import com.aliucord.annotations.AliucordPlugin;
import com.aliucord.entities.Plugin;
import com.discord.widgets.chat.list.adapter.WidgetChatListAdapterItemMessage;

@AliucordPlugin
public class NoLinkify extends Plugin {
    @Override
    public void start(Context context) {
        patcher.instead(WidgetChatListAdapterItemMessage.class, "shouldLinkify", (callFrame) -> false, String.class);
    }

    @Override
    public void stop(Context context) {
        patcher.unpatchAll();
    }
}

File: plugin/NoLinkify/src/main/kotlin/NoLinkify.kt
import android.content.Context
import com.aliucord.annotations.AliucordPlugin
import com.aliucord.entities.Plugin
import com.aliucord.patcher.instead
import com.discord.widgets.chat.list.adapter.WidgetChatListAdapterItemMessage

@AliucordPlugin
class NoLinkify : Plugin() {
    override fun start(context: Context) {
        patcher.instead<WidgetChatListAdapterItemMessage>("shouldLinkify", String::class.java) {
            false
        }
    }

    override fun stop(context: Context) = patcher.unpatchAll()
}